src/client/Microsoft.Identity.Client/Extensibility/AbstractConfidentialClientAcquireTokenParameterBuilderExtension.cs

```diff
@@ -53,13 +58,37 @@ public static AbstractAcquireTokenParameterBuilder<T> WithProofOfPosessionKeyId<
             }
 
             builder.ValidateUseOfExperimentalFeature();
-            builder.CommonParameters.AuthenticationScheme = new ExternalBoundTokenScheme(keyId, expectedTokenTypeFromAad);
+            builder.CommonParameters.AuthenticationOperation = new ExternalBoundTokenScheme(keyId, expectedTokenTypeFromAad);
 
             return builder;
         }
-
 #if !MOBILE
         /// <summary>
+        /// Enables client applications to provide a custom authentication operation to be used in the token acquisition request.
+        /// <param name="builder">The builder to chain options to</param>
+        /// <param name="authenticationExtension">The implementation of the authenticaiton operation.</param>
+        /// <returns></returns>
+        public static AbstractAcquireTokenParameterBuilder<T> WithAuthenticationExtension<T>(
```

Comment:
Please make it experimental API. We don't want ppl to take dependency on this, we want this only for Id.Web